src/cpp/client/channel_cc.cc

```diff
@@ -175,25 +192,24 @@ class ChannelConnectivityWatcher {
   gpr_thd_id thd_id_;
   CompletionQueue cq_;
   gpr_refcount ref_;
+  int channel_count_;
 
-  static gpr_once g_connectivity_watcher_once_;
-  static gpr_mu g_watcher_mu_;
-  // protected under g_watcher_mu_
-  static ChannelConnectivityWatcher* g_watcher_;
+  std::mutex shutdown_mu_;
+  std::condition_variable shutdown_cv_;  // protected by shutdown_cv_
```

Comment:
The comment here should say `shutdown_mu_` instead of `shutdown_cv_`.

Same thing on the next line.

Reply:
Fixed.